unlock-app/src/constants.js

```diff
@@ -62,7 +62,7 @@ export const MONTH_NAMES = [
 ]
 
 // constants related to pagination
-export const PGN_ITEMS_PER_PAGE = 10
+export const PGN_ITEMS_PER_PAGE = 5
```

Comment:
I think 5 is probably safer for now given the poor performance of our listing API...  Also looks better in the UI IMHO.